src/averageAges.js

```diff
@@ -37,7 +50,28 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => person.sex === 'f');
+  const womenWithChildren = findWomenWithChildren(people);
```

Comment:
reason of moving this into function? you can just filter `women` array with `people.find(...)`